pkg/api/helper/helpers.go

```diff
@@ -265,6 +265,11 @@ func IsIntegerResourceName(str string) bool {
 	return integerResources.Has(str) || IsExtendedResourceName(api.ResourceName(str))
 }
 
+// Extended and Hugepages resources
+func IsScalarResourceName(name api.ResourceName) bool {
```

Comment:
Do we need this function here?

Reply:
In my understanding, helper/helpers.go and v1/helper/helpers.go have been kept in sync previously, Isn't that case any more?

Reply:
They don't look exactly in sync, but if that's the convention, it is fine.